app/moon.yml

```diff
@@ -0,0 +1,24 @@
+language: 'javascript'
+
+type: 'application'
+
+toolchain:
+    node:
+        version: '24'
```

Comment:
Do we need to specify it alse here as it is already defined in `.moon/toolchain.yml` ?

Reply:
You're right, it's not needed in each application.
 As long, we have defined it in `.moon/toolchain.yml` , moon will infer the needed version. 
 This can be helpful if need to manager multiple version of a language/runtime (not our case)